Compile error with GCC 15

**Describe the bug**
Building QLC+ fails when building with GCC15 (tested on Fedora Rawhide).

**To Reproduce**
Build QLC+ for QT-6 as documented and observe the following build error:
```
make[2]: Entering directory '/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1'
[  9%] Building CXX object plugins/E1.31/CMakeFiles/e131.dir/e131controller.cpp.o
cd /builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31 && /usr/bin/g++ -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DUNICODE -De131_EXPORTS -I/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31 -I/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/e131_autogen/include -I/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/../interfaces -isystem /usr/include/qt6/QtCore -isystem /usr/include/qt6 -isystem /usr/lib64/qt6/mkspecs/linux-g++ -isystem /usr/include/qt6/QtGui -isystem /usr/include/qt6/QtNetwork -isystem /usr/include/qt6/QtWidgets -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -march=x86-64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -mtls-dialect=gnu2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -Werror -Wextra -Wall -DNDEBUG -fPIC -MD -MT plugins/E1.31/CMakeFiles/e131.dir/e131controller.cpp.o -MF CMakeFiles/e131.dir/e131controller.cpp.o.d -o CMakeFiles/e131.dir/e131controller.cpp.o -c /builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/e131controller.cpp
In file included from /builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/e131controller.cpp:20:
/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/e131controller.h:43:32: error: field ‘inputSocket’ has incomplete type ‘QSharedPointer<QUdpSocket>’
   43 |     QSharedPointer<QUdpSocket> inputSocket;
      |                                ^~~~~~~~~~~
In file included from /usr/include/qt6/QtCore/qobject.h:18,
                 from /usr/include/qt6/QtCore/qiodevice.h:10,
                 from /usr/include/qt6/QtNetwork/qabstractsocket.h:14,
                 from /usr/include/qt6/QtNetwork/qhostaddress.h:12,
                 from /usr/include/qt6/QtNetwork/qnetworkinterface.h:10,
                 from /usr/include/qt6/QtNetwork/QNetworkInterface:1,
                 from /builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/e131controller.h:27:
/usr/include/qt6/QtCore/qmetatype.h:871:1: note: declaration of ‘class QSharedPointer<QUdpSocket>’
  871 | QT_FOR_EACH_AUTOMATIC_TEMPLATE_SMART_POINTER(QT_FORWARD_DECLARE_SHARED_POINTER_TYPES_ITER)
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/e131controller.h:163:32: error: field ‘m_UdpSocket’ has incomplete type ‘QSharedPointer<QUdpSocket>’
  163 |     QSharedPointer<QUdpSocket> m_UdpSocket;
      |                                ^~~~~~~~~~~
/usr/include/qt6/QtCore/qmetatype.h:871:1: note: declaration of ‘class QSharedPointer<QUdpSocket>’
  871 | QT_FOR_EACH_AUTOMATIC_TEMPLATE_SMART_POINTER(QT_FORWARD_DECLARE_SHARED_POINTER_TYPES_ITER)
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/e131controller.h:170:32: error: field ‘m_dmxValuesMap’ has incomplete type ‘QMap<unsigned int, QByteArray*>’
  170 |     QMap<quint32, QByteArray*> m_dmxValuesMap;
      |                                ^~~~~~~~~~~~~~
In file included from /usr/include/qt6/QtCore/qtypeinfo.h:9,
                 from /usr/include/qt6/QtCore/qglobal.h:45,
                 from /usr/include/qt6/QtCore/qshareddata.h:7,
                 from /usr/include/qt6/QtNetwork/qnetworkinterface.h:8:
/usr/include/qt6/QtCore/qcontainerfwd.h:23:43: note: declaration of ‘class QMap<unsigned int, QByteArray*>’
   23 | template <typename Key, typename T> class QMap;
      |                                           ^~~~
/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1/plugins/E1.31/e131controller.h:174:33: error: field ‘m_universeMap’ has incomplete type ‘QMap<unsigned int, _uinfo>’
  174 |     QMap<quint32, UniverseInfo> m_universeMap;
      |                                 ^~~~~~~~~~~~~
/usr/include/qt6/QtCore/qcontainerfwd.h:23:43: note: declaration of ‘class QMap<unsigned int, _uinfo>’
   23 | template <typename Key, typename T> class QMap;
      |                                           ^~~~
make[2]: *** [plugins/E1.31/CMakeFiles/e131.dir/build.make:171: plugins/E1.31/CMakeFiles/e131.dir/e131controller.cpp.o] Error 1
make[2]: Leaving directory '/builddir/build/BUILD/qlcplus-QLC+_4.14.1-build/qlcplus-QLC-_4.14.1'
make[1]: *** [CMakeFiles/Makefile2:8906: plugins/E1.31/CMakeFiles/e131.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

**Expected behavior**
A successful build.

**Desktop (please complete the following information):**
 - OS: Fedora Rawhide (x86-64 and aarch64)
 - QLC+ Version 4.14.1

I haven't debugged this yet, but from a first look at the error, I believe this is either a missing include (that was not necessary before) or a bug in QT-6.

--- a/plugins/E1.31/e131controller.h
+++ b/plugins/E1.31/e131controller.h
@@ -22,8 +22,10 @@
 
 #if defined(ANDROID)
 #include <QScopedPointer>
-#include <QSharedPointer>
 #endif
+#include <QByteArray>
+#include <QMap>
+#include <QSharedPointer>
 #include <QNetworkInterface>
 #include <QHostAddress>
 #include <QUdpSocket>

